config/enabled.yml

```diff
@@ -1279,6 +1279,12 @@ Rails/ReadWriteAttribute:
                  write_attribute(:attr, val).
   Enabled: true
 
+Rails/SafeNavigationOperator:
+  Description: 'Prefer `&.` over `ActiveSupport#try!`.'
+  StyleGuide: 'https://github.com/bbatsov/rails-style-guide#try-bang'
+  CaptureTry: false
```

Comment:
Since we can't safely identify the cases when `try` is equivalent to `&.` in static analysis, I suggest you remove the configuration option and report the uses of `try` that we know work the same as `try!`, i.e. calls with no arguments and a block.